src/app/modules/ledger/optin/cosigOptinDTOLedger.js

```diff
@@ -44,15 +44,15 @@ exports.CosigOptinDTOLedger = CosigOptinDTOLedger;
  * @param convertDTO
  * @param multisigDestination
  */
-CosigOptinDTOLedger.createLedger = async (cosigner, cosignerAcountPath, convertDTO, multisigDestination, network) => {
+CosigOptinDTOLedger.createLedger = async (cosigner, cosignerAccountPath, convertDTO, multisigDestination, network) => {
     const transaction = symbol_sdk_1.TransactionMapping.createFromPayload(convertDTO.p);
     if (transaction instanceof symbol_sdk_1.AggregateTransaction) {
         const cosignatureTransaction = symbol_sdk_1.CosignatureTransaction.create(transaction);
         let signature;
         const ledgerService = new LedgerService();
         if (cosigner.privateKey === undefined) {
-            const result = await ledgerService.signSymbolTransaction(cosignerAcountPath, cosignatureTransaction.transactionToCosign, constants_1.OptinConstants[network].CATAPULT_GENERATION_HASH, cosigner.publicKey);
-            signature = result.hash;
+            const result = await ledgerService.signSymbolAggregateTransaction(cosignerAccountPath, cosignatureTransaction.transactionToCosign, constants_1.OptinConstants[network].CATAPULT_GENERATION_HASH, cosigner.publicKey);
```

Comment:
should this just be the `convertDTO.h` ?
why pass the whole aggregate transaction for consigning?